Dockerfile

```diff
@@ -38,29 +38,30 @@ RUN apt-get update && \
 
 RUN pip install --no-cache-dir pip==20.2.3
 
+# Update all needed tool versions here
+
 ARG AWS_IAM_AUTHENTICATOR_VERSION=1.17.9
```

Comment:
this can likely be updated as well, but we can also skip it if we don't want to do it in this PR